Jupiter

```diff
@@ -3,4 +3,5 @@ Jupiter 67 moons: Ganymede Callisto Io Europa
 The planets equatorial clouds take 18 hours to complete one rotation. The
 reason this happens is that Neptune does not have a solid body.
 
+These violent spots have violent ends.
```

Comment:
I was really confused as how this was prophetic, at first, before re-reading to realize you said _violent_, not violet...